src/app/components/ATIAnalytics/canonical/staticBeacon/index.ts

```diff
@@ -0,0 +1,14 @@
+export const addStaticBeacon = () => `
```

Comment:
The names of the 2 exports are quite similar, and I wonder if there's a better way to differentiate... Something along these lines (but let's get a second opinion from another reviewer)

```suggestion
export const defineSendStaticBeacon = () => `
```
```suggestion
export const registerSendStaticBeacon = () => `
```
```suggestion
export const configureSendStaticBeacon = () => `
```
```suggestion
export const addSendStaticBeaconToWindow = () => `
```